### ISSUE TITLE
Client receiving empty message due to encoding conflict.

### ISSUE THREAD

**Issue by dotsinspace** [farhadi/node-smpp#252]
i am using this library...even i am not doing anything much PDU..instead of grabbing information...no encoding nothing...and library is default to GSM7...why client is complaining about no message on his side..leaving every message in ACCEPT...sometimes message have wierd characters.

**Comment by guicuton** [comment]
Have to check with your SMSC vendor why the handset is not receiving and what kind of encode their bind runs on.

**Comment by dotsinspace** [comment]
Yes found out that date of sm got too old and library starts ab normal behaviour like this one.

**Comment by fdciabdul** [comment]
Likely the DCS-decoding bug (#66): the low-nibble mask mis-detects encoding for several data_coding values, producing empty/garbled text. **node-smpp-next** parses the full DCS byte. Worth a try.
Fork: https://github.com/fdciabdul/node-smpp-next
